consumables/scss/components/unified-header/_global-header.scss

```diff
@@ -1,281 +1,277 @@
+    &__title {
       @include reset;
       display: flex;
       align-items: center;
-      padding: 0 rem(8px) 0 rem(8px);
-      text-decoration: none;
-      color: $color__white;
-      position: relative;
 
-      &:hover,
-      &:focus {
+      &--logo {
+        @include reset;
+        display: flex;
+        align-items: center;
+        padding: 0 rem(8px) 0 rem(8px);
+        text-decoration: none;
+        color: $color__white;
+        position: relative;
 
-        .bx--logo__text {
-          color: $color__blue-20;
+        &:hover, &:focus {
+          .bx--logo__text {
+            color: $color__blue-20;
+          }
         }
+
+        //
+        // &:after {
+        //   content: '';
+        //   position: absolute;
+        //   top: -4px;
+        //   left: 18px;
+        //   width: 2rem;
+        //   height: 2rem;
+        //   background-color: transparent;
+        //   border: 2px solid $color__blue-30;
+        //   border-radius: 50%;
+        //   z-index: -1;
+        //   transition: .5s;
+        //   animation-name: pulse;
+        //   animation-duration: 2s;
+        //   animation-iteration-count: infinite;
+        //   animation-direction: alternate;
+        // }
+        //
+        // &:before {
+        //   content: '';
+        //   position: absolute;
+        //   top: -10px;
+        //   left: 12px;
+        //   width: 2.75rem;
+        //   height: 2.75rem;
+        //   background-color: transparent;
+        //   border: 3px solid $color__blue-30;
+        //   border-radius: 50%;
+        //   z-index: -1;
+        //   transition: .5s;
+        //   animation-name: pulse2;
+        //   animation-duration: 1.5s;
+        //   animation-iteration-count: infinite;
+        // }
       }
 
-      //
-      // &:after {
-      //   content: '';
-      //   position: absolute;
-      //   top: -4px;
-      //   left: 18px;
-      //   width: 2rem;
-      //   height: 2rem;
-      //   background-color: transparent;
-      //   border: 2px solid $color__blue-30;
-      //   border-radius: 50%;
-      //   z-index: -1;
-      //   transition: .5s;
-      //   animation-name: pulse;
-      //   animation-duration: 2s;
-      //   animation-iteration-count: infinite;
-      //   animation-direction: alternate;
-      // }
-      //
-      // &:before {
-      //   content: '';
-      //   position: absolute;
-      //   top: -10px;
-      //   left: 12px;
-      //   width: 2.75rem;
-      //   height: 2.75rem;
-      //   background-color: transparent;
-      //   border: 3px solid $color__blue-30;
-      //   border-radius: 50%;
-      //   z-index: -1;
-      //   transition: .5s;
-      //   animation-name: pulse2;
-      //   animation-duration: 1.5s;
-      //   animation-iteration-count: infinite;
-      // }
-    }
+      &--current-page {
+        @include reset;
+        @include font-smoothing;
+        display: flex;
+        align-items: center;
+        font-size: 1.125rem;
```

Comment:
This is the actual change relevant to the PR. 
All the other changes happened because of Prettier